storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2664,6 +2721,12 @@ static bool xtrabackup_read_metadata(char *filename) {
       1) {
     backup_redo_log_flushed_lsn = 0;
   }
+  if (fscanf(fp, "redo_memory = " LSN_PF "\n", &redo_memory) != 1) {
+    redo_memory = 0;
+  }
+  if (fscanf(fp, "redo_frames = " LSN_PF "\n", &redo_frames) != 1) {
```

Comment:
Can you pleas fix this warning warning: format specifies type 'unsigned long long *' but the argument has type 'ulint *' (aka 'unsigned long *') [-Wformat]
  if (fscanf(fp, "redo_frames = " LSN_PF "\n", &redo_frames) != 1) {